.golangci.yml

```diff
@@ -13,6 +13,29 @@ linters:
       - lll
       - errcheck
       - interfacer
+      - tagliatelle
+      - varnamelen
+      - nonamedreturns
+      - nlreturn # re-enable this after fixing all cases
+      - gomnd # re-enable this after fixing all cases
+      - nosnakecase
+      - ifshort # re-enable
+      - nilerr #re-enable
+      - goerr113
+      - exhaustivestruct
+      - exhaustruct
+      - exhaustive
+      - godot
+      - wsl #fix
+      - gocognit
+      - revive
+      - errname
+      - cyclop
+      - gofumpt # fix
+      - gci
+      - nestif # fix
+      - gosec
+      - gocritic
```

Comment:
updated version of golangci-lint and had to disable some rules. we can revisit this later as KTLO to do lint fixes and re-enable these rules